docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_remove_util_deps.txt

```diff
@@ -0,0 +1 @@
+* Deprecated functions ```iris.util.broadcast_weights``` and ```iris.util.ensure_array``` have been removed from :mod:`iris.util`.
```

Comment:
@corinnebosley you've missed `iris.util.timers` (see https://github.com/SciTools/iris/compare/v1.13.x...master?expand=1#diff-b6f4583aa861d9305fec9d55d9c18dfaL987), which is how the private class `_Timers()` is publically exposed...

Reply:
Oh I see... let me just fix that then.